lib/workers/global/index.js

```diff
@@ -75,6 +75,16 @@ async function start() {
       logger.fatal({ err }, `Fatal error: ${err.message}`);
     }
   }
+  const loggerErrors = getErrors();
+  /* istanbul ignore if */
+  if (loggerErrors.length) {
+    logger.info(
+      { loggerErrors },
```

Comment:
Do we need to worry about sanitizing anything here? e.g. if the metadata includes a `config` key then it will be sanitized but that might be missed here?

Reply:
I can't think of any simple way to achieve this though

Reply:
🤔  have to look how `bunyan` works here

Reply:
what do you think about using a `stream` and store the `BunyanRecord` records, they should already be sanitized

Reply:
Sounds ok if you think you can get it to work